RecoLocalCalo/EcalRecProducers/plugins/alpaka/EcalUncalibRecHitPhase2WeightsAlgoPortable.dev.cc

```diff
@@ -0,0 +1,111 @@
+#include "DataFormats/EcalDigi/interface/EcalDataFrame_Ph2.h"
+#include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
+
+#include "EcalUncalibRecHitPhase2WeightsAlgoPortable.h"
+
+#include "DataFormats/EcalDigi/interface/alpaka/EcalDigiPhase2DeviceCollection.h"
+#include "DataFormats/EcalRecHit/interface/alpaka/EcalUncalibratedRecHitDeviceCollection.h"
+#include <alpaka/alpaka.hpp>
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+#include "DataFormats/EcalRecHit/interface/EcalUncalibratedRecHit.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  namespace ecal {
+    namespace weights {
```

Comment:
These could be expressed as 
```suggestion
namespace ALPAKA_ACCELERATOR_NAMESPACE::ecal::weights {
```
in order to have shorter indentation in the code below.